File: cm-backend/src/main/java/com/clothing/management/dto/PacketValidationDTO.java
package com.clothing.management.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PacketValidationDTO {
    private Long id;
    private String customerName;
    private String customerPhoneNb;
    private String packetDescription;
    private String barcode;
    private String fbPageName;
    private String deliveryCompanyName;
    private double price;
    private Date date;
    private boolean valid;
    private Long exchangeId;
    private boolean haveExchange;
}

File: cm-front/src/shared/models/PacketValidationDTO.ts
export interface PacketValidationDTO {
    id: number;
    date: Date;
    customerName: string;
    packetDescription: string;
    customerPhoneNb: string;
    barcode: any;
    fbPage: string;
    deliveryCompanyName: string;
    price: number;
    valid: boolean;
    exchangeId: number;
    haveExchange:boolean;
}
